### DISCUSSION TITLE
Add ignore_path parameter for ESLint task

### DISCUSSION THREAD

**Discussion by tom-combet** [phpro/grumphp#961] [Ideas]
| Q               | A
| --------------- | ---
| Version         | 1.5.0
| Bug?            | no
| New feature?    | yes
| Question?       | no
| Documentation?  | no
| Related tickets | none

GrumPHP should support a custom eslintignore path parameter for eslint task.
My eslintrc.js and .eslintignore files are in a subfolder. Config file is recognized but not the ignore one. If I put .eslintignore file at the root of the project, it's recognized.
The new task parameter could be called `ignore_path` and would use `--ignore-path` eslint command line option.

**Comment by veewee** [comment]
I'dd surely accept a PR for that. Care to give it a try @Kaudaj ?

**Reply by tom-combet** [reply]
Ok I accept the mission! 😎
By the way, do you prefer I create directly a discussion instead of an issue next time?

**Reply by veewee** [reply]
Sure, let me know if you have any questions.

We use the issues section for tracking real bugs and discussions for q/a or feature requests.
